cfpb/cfgov-refresh
# Fix for GTM ignore attribute being applied on desktop

Fix for GTM ignore attribute being applied on desktop


## Changes

- Modified `cfgov/unprocessed/js/modules/behavior/FlyoutMenu.js` to fix issue with GTM ignore code being applied on desktop.

## Testing

- Visit that site on desktop and verify the GTM ignore code isn't applied on the `Educational Resources` navigation item. 
- Visit that site on mobile and verify the GTM ignore code is applied on the `Educational Resources` navigation item. 

## Screenshots


## Notes

- I'm not adding resize code which would detect a change from desktop to mobile. Just don't think it's necessary.

## Todos

-

## Checklist
